src/Shared/Recruit.Vacancies.Client/Infrastructure/Events/QueueNames.cs

```diff
@@ -4,7 +4,7 @@ public static class QueueNames
     {
         public const string VacancyEventsQueueName = "vacancy-events-queue";
         public const string VacancyReviewEventsQueueName = "vacancy-review-events-queue";
-        public const string DashboardQueueName = "dashboard-queue";
+        public const string EmployerDashboardQueueName = "employer-dashboard-queue";
```

Comment:
Would you mind prefixing this queue name with `generate-`. This was my fault for giving it a bad name in the first place.

Reply:
Done